media-libs/opencv/opencv-3.2.0-r1.ebuild

```diff
@@ -131,6 +162,11 @@ src_configure() {
 	export ANT_OPTS+=" -Dfile.encoding=iso-8859-1"
 	java-ant-2_src_configure
 
+	ENABLE_OMIT_FRAME_POINTER=OFF
+	if [ ! -z $(get-flag -fomit-frame-pointer) ]; then
```

Comment:
You have to inherit flag-o-matic eclass if you use the function from that eclass. Why not [[ ]] ? This whole piece of code is shaky..